samza-core/src/test/java/org/apache/samza/container/TestRunLoop.java

```diff
@@ -538,6 +533,9 @@ public void testEndOfStreamWithMultipleTasks() throws Exception {
     task0ProcessedMessagesLatch.await();
     task1ProcessedMessagesLatch.await();
 
+    verify(task0, times(1)).endOfStream(any());
```

Comment:
`times(1)` is the default for `verify`, so you don't need to pass as an argument.

Reply:
Thanks, didn't know this.